### ISSUE TITLE
Dilatation of a bronchial artery 

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6526]
**Preferred term label:** Dilatation of a bronchial artery 

**Synonyms**
Bronchial artery enlargement is recognised when its caliber exceeds 2 mm.

**Definition (free text, please give PubMed ID)**
Bronchial artery enlargement

**Parent term (use hpo.jax.org/app)**
Abnormality of bronchial artery (to be created)

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**

Idiopathic pulmonary arterial hypertension ORPHA:275766
Hereditary pulmonary arterial hypertension ORPHA:275777
Pulmonary arterial hypertension associated with congenital heart disease

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
adding term together with this citation https://pubmed.ncbi.nlm.nih.gov/32430593/
